diff --git a/node_modules/@mintplex-labs/piper-tts-web/dist/piper-tts-web.js b/node_modules/@mintplex-labs/piper-tts-web/dist/piper-tts-web.js
index 32767f4..ff5a7cf 100644
--- a/node_modules/@mintplex-labs/piper-tts-web/dist/piper-tts-web.js
+++ b/node_modules/@mintplex-labs/piper-tts-web/dist/piper-tts-web.js
@@ -9,7 +9,7 @@ var __privateGet = (obj, member, getter) => (__accessCheck(obj, member, "read fr
 var __privateAdd = (obj, member, value) => member.has(obj) ? __typeError("Cannot add the same private member more than once") : member instanceof WeakSet ? member.add(obj) : member.set(obj, value);
 var __privateSet = (obj, member, value, setter) => (__accessCheck(obj, member, "write to private field"), setter ? setter.call(obj, value) : member.set(obj, value), value);
 var _createPiperPhonemize, _modelConfig, _ort, _ortSession, _progressCallback, _wasmPaths, _logger;
-const HF_BASE = "https://huggingface.co/diffusionstudio/piper-voices/resolve/main";
+const HF_BASE = "/piper";
 const ONNX_BASE = "https://cdnjs.cloudflare.com/ajax/libs/onnxruntime-web/1.18.0/";
 const WASM_BASE = "https://cdn.jsdelivr.net/npm/@diffusionstudio/piper-wasm@1.0.0/build/piper_phonemize";
 const PATH_MAP = {
@@ -47,7 +47,7 @@ const PATH_MAP = {
   "en_US-amy-medium": "en/en_US/amy/medium/en_US-amy-medium.onnx",
   "en_US-arctic-medium": "en/en_US/arctic/medium/en_US-arctic-medium.onnx",
   "en_US-danny-low": "en/en_US/danny/low/en_US-danny-low.onnx",
-  "en_US-hfc_female-medium": "en/en_US/hfc_female/medium/en_US-hfc_female-medium.onnx",
+  "en_US-hfc_female-medium": "en_US-hfc_female-medium.onnx",
   "en_US-hfc_male-medium": "en/en_US/hfc_male/medium/en_US-hfc_male-medium.onnx",
   "en_US-joe-medium": "en/en_US/joe/medium/en_US-joe-medium.onnx",
   "en_US-kathleen-low": "en/en_US/kathleen/low/en_US-kathleen-low.onnx",
@@ -140,7 +140,7 @@ const PATH_MAP = {
   "it_IT-paola-medium": "it/it_IT/paola/medium/it_IT-paola-medium.onnx"
 };
 async function writeBlob(url, blob) {
-  if (!url.match("https://huggingface.co")) return;
+  
   try {
     const root = await navigator.storage.getDirectory();
     const dir = await root.getDirectoryHandle("piper", {
@@ -167,7 +167,7 @@ async function removeBlob(url) {
   }
 }
 async function readBlob(url) {
-  if (!url.match("https://huggingface.co")) return;
+  
   try {
     const root = await navigator.storage.getDirectory();
     const dir = await root.getDirectoryHandle("piper", {
